lang/cs/Org.Apache.REEF.Client/Common/JobSubmissionResult.cs

```diff
@@ -96,6 +96,36 @@ public string GetUrlResult(string url)
             return task.Result;
         }
 
+        public void WaitForDriverToFinish()
+        {
+            DriverStatus status = FetchDriverStatus();
+            
+            while (status.IsActive())
+            {
+                try
+                {
+                    status = FetchDriverStatus();
+                }
+                catch (System.Net.WebException)
+                {
+                    // If we no longer can reach the Driver, it must have exited.
+                    status = DriverStatus.UNKNOWN_EXITED;
+                }
+            }
+        }
+
+        private DriverStatus FetchDriverStatus()
+        {
+            string statusUrl = DriverUrl + "driverstatus/v1";
+            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(statusUrl);
+            using (StreamReader reader = new StreamReader(request.GetResponse().GetResponseStream()))
+            {
+                string statusString = reader.ReadToEnd();
+                LOGGER.Log(Level.Verbose, "Status received: " + statusString);
```

Comment:
use string interpolation

Reply:
When I use the `$something{here}` syntax, VS complains. I assume we are not setup for the newest version of C#? Shall I use `string.format` instead?

Reply:
We shall never use `string.Format()` for logging. Use `{0}` placeholders instead, e.g.
```C#
LOGGER.Log(Level.Verbose, "Status received: {0}", statusString);
```

Reply:
Thanks, fixed!